modules/ld/src/main/resources/resource-context.json

```diff
@@ -12,19 +12,19 @@
     "skos": "http://www.w3.org/2004/02/skos/core#",
```

Comment:
where did the `config` and `maxAttachmentSize` went?

Reply:
I thought that we decided to drop "maxAttachmentSize" from projects for now. Thought @bogdanromanx